join.html

```diff
@@ -207,7 +207,9 @@ <h2 class="progress">Step 1 of 4: Personal Details</h2>
             type="text"
             name="country"
             id="country"
-          ></select>
+          >
+            <option value="" disabled selected>e.g India</option>
+          </select>
```

Comment:
This Part of the code is not working, I can not see anything already selected

![image](https://user-images.githubusercontent.com/57758447/209892602-3e19e213-3801-4ad0-b3c7-729f07230d79.png)

Reply:
@RitikJaiswal75 It is working fine please have a look
<img width="626" alt="image" src="https://user-images.githubusercontent.com/74719713/210062378-602a41f6-1d3a-4187-9c05-886fb76d84cc.png">

Reply:
@Neenu-12 Please clear you local storage and then check for default selection

Reply:
Hi @rohan09-raj  cleared the local storage as you suggested and here's the screenshot of the page :-

<img width="482" alt="image" src="https://user-images.githubusercontent.com/74719713/211620679-3abc86cd-386c-42a7-ba9f-d2b2e5898ef3.png">

It's working fine

Reply:
Great thanks 💯